packages/analytics-react-native/README.md

```diff
@@ -33,6 +33,17 @@ Initialization is necessary before any instrumentation is done. The API key for
 amplitude.init(API_KEY);
 ```
 
+### Set UserId
+
+Sets an `userId` (usually called after user logs in).
+``typescript
+import { setUserId } from '@amplitude/analytics-react-native';
+
+// Track a basic event
```

Comment:
There is no tracking logic so maybe remove this comment?

Reply:
Done!